src/app/app.config.ts

```diff
@@ -0,0 +1,40 @@
+import { APP_INITIALIZER, ErrorHandler } from '@angular/core';
+import { Router } from '@angular/router';
+import * as Sentry from '@sentry/angular';
+import { environment } from 'src/environments/environment';
+if (environment.production) {
+  Sentry.init({
+    dsn: environment.SENTRY_DNS,
+    integrations: [
+      Sentry.browserTracingIntegration(),
+      Sentry.replayIntegration(),
+    ],
+    tracesSampleRate: 1.0,
+    tracePropagationTargets: ['localhost', /^https?:\/\/localhost:\d+/],
+    environment: environment.production
```

Comment:
we should have an env variable which would be called `SENTRY_ENV` it would have the string value of which environment it is